packages/Webkul/User/src/Models/Admin.php

```diff
@@ -8,8 +8,9 @@
 use Webkul\User\Contracts\Admin as AdminContract;
 use Webkul\User\Database\Factories\AdminFactory;
 use Webkul\User\Notifications\AdminResetPassword;
+use Tymon\JWTAuth\Contracts\JWTSubject;
```

Comment:
Tymon JWT auth is not maintained for the last 1.5 years, so this namespace is  used,
`use PHPOpenSourceSaver\JWTAuth\Contracts\JWTSubject;`